### ISSUE TITLE
Invariant Violation on simple example

### ISSUE THREAD

**Issue by SleeplessByte** [jxnblk/mdx-deck#353]
Given this example:

`package.json`
```json
{
  "scripts": {
    "start": "mdx-deck deck.mdx",
  },
  "devDependencies": {
    "mdx-deck": "^2.3.2"
  }
}
```

`deck.mdx`
```mdx
# Title
``` 

`error.log`
```
[err] { Invariant Violation: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
    at invariant (C:\Users\<UserName>\Documents\GitHub\<project>\node_modules\react\cjs\react.development.js:88:15)
    at resolveDispatcher (C:\Users\<UserName>\Documents\GitHub\<project>\node_modules\react\cjs\react.development.js:1436:28)
    at useContext (C:\Users\<UserName>\Documents\GitHub\<project>\node_modules\react\cjs\react.development.js:1441:20)
    at Rn (C:\Users\<UserName>\Documents\GitHub\<project>\dist\__app.js:14:71010)
    at Mn (C:\Users\<UserName>\Documents\GitHub\<project>\dist\__app.js:14:71188)
    at processChild (C:\Users\<UserName>\Documents\Github\<project>\node_modules\react-dom\cjs\react-dom-server.node.development.js:2888:14)
    at resolve (C:\Users\<UserName>\Documents\Github\<project>\node_modules\react-dom\cjs\react-dom-server.node.development.js:2812:5)
    at ReactDOMServerRenderer.render (C:\Users\<UserName>\Documents\Github\<project>\node_modules\react-dom\cjs\react-dom-server.node.development.js:3202:22)
    at ReactDOMServerRenderer.read (C:\Users\<UserName>\Documents\Github\<project>\node_modules\react-dom\cjs\react-dom-server.node.development.js:3161:29)
    at renderToString (C:\Users\<UserName>\Documents\Github\<project>\node_modules\react-dom\cjs\react-dom-server.node.development.js:3646:27)
    at renderHTML (C:\Users\<UserName>\Documents\Github\<project>\node_modules\mdx-deck\lib\build.js:41:16)
    at build (C:\Users\<UserName>\Documents\Github\<project>\node_modules\mdx-deck\lib\build.js:64:34)
    at processTicksAndRejections (internal/process/task_queues.js:86:5) name: 'Invariant Violation', framesToPop: 1 }
```

I am running on node 11, Windows 10; does not seem related to the other Windows bug(s).

By a load-race trick, I was able to launch the dev server and then make it error. It gives the same issue, in addition to:

```
The above error occurred in the <MDXCreateElement> component:
    in MDXCreateElement
    in MDXContent (created by UserHead)
    in UserHead (created by Provider)
    in HeadProvider (created by Provider)
    in Provider (created by MDXDeck)
    in MDXDeck (created by App)
    in App
```

**Comment by techieshark** [comment]
I had a perhaps similar issue with an error about an "Invariant Violation" and react version, and this comment helped:

https://github.com/jxnblk/mdx-deck/issues/125#issuecomment-415475360

tl;dr: remove `node_modules` directory and lock file, then npm install again.

**Comment by SleeplessByte** [comment]
@techieshark I'll try and report back!
